app/scripts/lib/app-start.js

```diff
@@ -221,6 +210,12 @@ function (
       this._configLoader.useConfig(config);
     },
 
+    initializeErrorMetrics: function () {
+      if (this._config && this._config.sentrySampleRate && isMetricsCollectionEnabled(this._config.sentrySampleRate)) {
```

Comment:
with your work to use Able to determine whether the user is part of the "collect metrics" bucket, do you think we should use Able for Sentry errors as well?